src/scenes/home/signup/signup.js

```diff
@@ -81,10 +82,12 @@ class SignUp extends Component {
           identifier
         }
       }).then(({ data }) => {
+        console.log('Success!');
         window.location = '/signup-info';
         CookieHelpers.setUserAuthCookie(data);
         this.setState({ isLoading: false });
       }).catch((error) => {
+        console.log('Failed!');
```

Comment:
@PixelWitch Any reason these console.logs are left here, and for disabling the `no-console` rule in ESLint? Seems like they would be useful during development, but shouldn't be in production.

Reply:
Hi John, I've removed the .jshintrc and .jshintignore files. The console logs are there because the social logins are still undergoing testing and bugfixing, and we use the console for that; In the final build these will be removed and the rule will be turned back on. Currently this pull request is here for review purposes. Thanks for your feedback, and let me know if you need anything else changed or have any more questions.